features/importers/venepaikka_harbors/importer.py

```diff
@@ -0,0 +1,213 @@
+from features.models import ContactInfo, Feature, Image, License, SourceType, Tag
+
+query = """
+query Harbors {
+  harbors {
+    edges {
+      node {
+        id
+        geometry {
+          type
+          coordinates
+        }
+        properties {
+          name
+          wwwUrl
+          imageLink
+          imageFile
+          streetAddress
+          zipCode
+          municipality
+          phone
+          email
+          servicemapId
+        }
+      }
+    }
+  }
+}
+"""
+
+feature_expression = jmespath.compile(
+    """
+data.harbors.edges[*].node.{
+    id: id,
+    name: properties.name,
+    url: properties.wwwUrl,
+    lat: geometry.coordinates[1],
+    lon: geometry.coordinates[0],
+    address: {
+        street_address: properties.streetAddress,
+        postal_code: properties.zipCode,
+        municipality: properties.municipality
+        phone_number: properties.phone
+        email: properties.email
+    },
+    images: [properties.imageFile, properties.imageLink]
+    servicemap_id: properties.servicemapId
+}
+"""
+)
+
+
+class VenepaikkaImporter(FeatureImporterBase):
+
+    source_system = "venepaikka"
+    source_type = "harbor"
+
+    servicemap_url = "https://palvelukartta.hel.fi/fi/unit/"
+    servicemap_link_type = "servicemap"
+    image_copyright_owner = "venepaikat.hel.fi"
+
+    def import_features(self):
+        source_type = self.get_source_type()
+        client = VenepaikkaHarborsClient()
+        harbors = client.fetch_harbors(query=query).json()
+
+        # Category, tag and image license is the same for all harbours
+        category, created = Category.objects.language("fi").update_or_create(
+            id=app_settings.CATEGORY_CONFIG["id"],
+            defaults={"name": app_settings.CATEGORY_CONFIG["name"]},
+        )
+        tag, created = Tag.objects.language("fi").update_or_create(
+            id=app_settings.TAG_CONFIG["id"],
+            defaults={"name": app_settings.TAG_CONFIG["name"]},
+        )
+        image_license = License.objects.translated(
+            "fi", name=app_settings.IMAGE_LICENSE
+        ).first()
+        if not image_license:
+            image_license = License.objects.language("fi").create(
+                name=app_settings.IMAGE_LICENSE
+            )
+
+        for harbor in feature_expression.search(harbors):
+            feature = self._import_feature(harbor, source_type)
+            self._set_feature_category(feature, category)
+            self._set_feature_tag(feature, tag)
+            self._import_feature_contact_info(feature, harbor["address"])
+            self._import_service_map_url(feature, harbor["servicemap_id"])
+            self._import_feature_images(feature, harbor["images"], image_license)
+
+    @staticmethod
+    def _import_feature(harbor: dict, st: SourceType) -> Feature:
+        values = {
+            "name": harbor["name"],
+            "url": harbor["url"],
```

Comment:
so, will you keep this imported for now?

Reply:
Ah, that.. I'll remove it for now.